packages/components/CHANGELOG.md

```diff
@@ -42,6 +42,10 @@
     -   `TreeSelect`
     -   `UnitControl`
 
+### Bug Fixes
+
+-   `Navigator`: fix isInitial & refine focus selector logic added in #64675 ([#64786](https://github.com/WordPress/gutenberg/pull/64786)).
```

Comment:
There wasn't a release since #64675 was merged, right? Then this changelog entry describes something that a package consumer never had a chance to experience, kind of redundant.

Reply:
Good point — I've removed the commit and currently this PR doesn't add any changelog entry.